### DISCUSSION TITLE
Feature-Request: Editing playlist

### DISCUSSION THREAD

**Discussion by djarcade** [regseb/castkodi#57] [Ideas]
hi 
great plug in and i use it all the time 
could you add a an playlist editing feature ?
so i can move an entry in the play list up and down

mock up picture how i image it 
https://imgur.com/a/yACZfk3

**Comment by regseb** [comment]
@djarcade In the new version _5.14.0_ (requires Firefox _89+_), you can edit playlist with drag-and-drop.
